app/views/posts/index.html.erb

```diff
@@ -1 +1 @@
-<h1>User Posts page</h1>
\ No newline at end of file
+<h1>User Posts page</h1>
```

Comment:
- [ ] Kindly implement the posts index page of a user. 

  - It should render the recent comments on the posts using the defined method in the model
  - Its layout should match the wireframe.

---

  your current post index page | wireframe for post index page
  :---:|:---:
  ![wey](https://user-images.githubusercontent.com/54848363/198067466-ae073324-7d8d-4bf8-a91c-436e6c994910.jpg) | ![blog_user_all_posts](https://user-images.githubusercontent.com/54848363/198067451-2282d3f9-06d4-4f47-96bf-e08f825ba4ae.jpg)

Reply:
Thank you @DammyShittu , I implemented it as required, for a better understanding of what I did check the 4th user, he is the one having a post and a lot of comments